### PR TITLE
Clean-up for the SPE10 Model 1

### PR DESCRIPTION
**Pull request by alfbr**
The fix of Bg values and adding a corner-point grid has been reintroduced. Also CPR is used rather than numerical tuning to improve convergence of linear solver (necessary for Eclipse). Unfortunately no reference run by OPM, but this we can revisit when time permits.

### CODE REVIEW

**Comment by alfbr** [comment]
Please consider reviewing @totto82

File: spe10model1/SPE10_MODEL1_CP.DATA [hunk 1]
```diff
@@ -0,0 +1,948 @@
...
+/
+
+WCONPROD
+'PROD'  OPEN   BHP    5*     95   /
+/
+
+WCONINJE
+'INJ'   GAS   OPEN  RESV  1*  43.839979 10000000  /
+/
+
```
Comment (totto82) [inline spe10model1/SPE10_MODEL1_CP.DATA:268]: Why is the control here different then in SPE10_MODEL1.DATA
Using the WCONINJE control from SPE10_MODEL1.DATA here is sufficient change for OPM to run SPE10_MODEL1_CP.DATA with identical results as Eclipse.

**Comment by atgeirr** [comment]
What is the status of this work?

**Comment by alfbr** [comment]
Still trying to allocate time to prioritize it. I will make an attempt this week.

**Comment by alfbr** [comment]
Case updated, thanks for the catch @totto82

**Comment by atgeirr** [comment]
Is this ready to be merged? I have not reviewed it.

**Comment by alfbr** [comment]
From my end yes, what do you say @totto82 ?

**Comment by totto82** [comment]
I belive the changes to the  .DATA file is ready to be merged. 
From the commits it looks like the eclipse simulation results are not updated.

**Comment by alfbr** [comment]
Indeed, again thanks for the catch. Reference simulations updated (again). I also can confirm that they match, at least as far as flow simulates until it fails to converge.

I did a bit of git-fu to rebase this branch, but is I did it right, it should be merge ready.

File: spe10model1/SPE10_MODEL1_CP.DATA [hunk 2]
```diff
@@ -0,0 +1,948 @@
...
+UNIFOUT
+
+START
+  1 'JAN' 1967 /
+
+
+--      =================================================
+------- In this section, the geometry of the simulation 
+------- grid and the rock perøeabilities, porosities are
```
Comment (pgdr) [inline spe10model1/SPE10_MODEL1_CP.DATA]: perøeabilities → permeabilities

**Comment by alfbr** [comment]
I will self-merge this tomorrow unless somebody comments.

**Comment by atgeirr** [comment]
Fine with me!

**Pull request merged**